plugins/janus_streaming.c

```diff
@@ -6445,7 +6449,7 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 		aresult = janus_streaming_rtsp_parse_sdp(curldata->buffer, name, "audio", abase, &apt,
 			atransport, ahost, artpmap, afmtp, acontrol, &source->audio_iface, &audio_fds);
 	}
-	janus_mutex_unlock(&mountpoints_mutex);
+	janus_mutex_unlock(&mp_mutex);
```

Comment:
There's no such thing as `mp_mutex` in the code.

Reply:
I will adjust it and  pull request again